opal/mca/common/verbs/common_verbs_basics.c

```diff
@@ -61,3 +64,30 @@ bool opal_common_verbs_check_basics(void)
     return true;
 }
 
+int opal_common_verbs_fork_test(void)
+{
+    /* Make sure that ibv_fork_init is called before the calls to other memory registering verbs,
+     * which will be called after this function */
+#ifdef HAVE_IBV_FORK_INIT
+    if (0 != opal_verbs_want_fork_support) {
+        /* Check if fork support is requested by the user */
+        if (0 != ibv_fork_init()) {
+            opal_show_help("help-opal-common-verbs.txt", "ibv_fork_init fail",
```

Comment:
Err... sorry @alinask -- there have been comments flying around out of order.  See @rolfv's later comments about changing the default value to 0 (vs. -1). And then my comments about "perhaps it should just be a boolean, not a tri-state".

Sorry for the confusion! :-(